integration/http.go

```diff
@@ -0,0 +1,28 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// MakeStandardRequest issues an HTTP request a specific endpoint with Atlas-specific
+// request data (e.g. the authorization token)
+func MakeStandardRequest(method, url string, payload interface{}) (*http.Response, error) {
```

Comment:
For now I see this only used in tests. What concerns me here is that we will create a new client for every request. Maybe it makes sense to allow client reuse by passing a client as a parameter. This may be useful as well when client needs some configuration (e.g. not just standard http.Client{}).

Reply:
Or as an option function may return `http.Request` instead of `http.Response`

Reply:
Good point. I see the problem with creating `http.Client` types over and over again. 

There's definitely room to improve. I like the idea of returning a `http.Request` type, so I'll start there.